### PR TITLE
Test (py-knights-fighting)

### PR DESCRIPTION
**Pull request by baranovr**

### CODE REVIEW

**Comment by baranovr** [comment]
Due to these errors, I had to change the original file:

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,68 @@
+from typing import List, Dict, Any
+
+
+class Knight:
+    def __init__(self,
```
Comment (Polyakiv-Andrey) [inline app/main.py:5]: After opening the parenthesis, write all the arguments on a new line, including the first one

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,68 @@
+from typing import List, Dict, Any
+
+
```
Comment (Polyakiv-Andrey) [inline app/main.py:3]: Break your code into multiple modules or files so everything is not in 1 heap

File: app/main.py [hunk 3]
```diff
@@ -86,121 +151,45 @@
...
-    if red_knight["potion"] is not None:
-        if "power" in red_knight["potion"]["effect"]:
-            red_knight["power"] += red_knight["potion"]["effect"]["power"]
-
-        if "protection" in red_knight["potion"]["effect"]:
-            red_knight["protection"] += red_knight["potion"]["effect"]["protection"]
-
-        if "hp" in red_knight["potion"]["effect"]:
-            red_knight["hp"] += red_knight["potion"]["effect"]["hp"]
+def battle(knights: Dict) -> Dict:
```
Comment (Polyakiv-Andrey) [inline app/main.py:170]: Add more annotations

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]

**Comment by baranovr** [comment]
Снимок экрана 2023-08-08 173238
Снимок экрана 2023-08-08 173245
Снимок экрана 2023-08-08 173252
Can you advice me: why I have this problem?

**Pull request closed without merging**